session.go

```diff
@@ -351,10 +359,11 @@ func (s *Session) mux(remoteBegin *performBegin) {
 
 				// if this message is received unsettled and link rcv-settle-mode == second, add to handlesByRemoteDeliveryID
 				if !body.Settled && body.DeliveryID != nil && link.receiverSettleMode != nil && *link.receiverSettleMode == ModeSecond {
-					debug(1, "TX: adding handle to handlesByRemoteDeliveryID. linkCredit: %d", link.linkCredit)
+					debug(1, "TX(Session): adding handle to handlesByRemoteDeliveryID. linkCredit: %d", link.linkCredit)
 					handlesByRemoteDeliveryID[*body.DeliveryID] = body.Handle
 				}
 
+				debug(3, "TX(Session) Flow? remoteOutgoingWindow(%d) < s.incomingWindow(%d)/2\n", remoteOutgoingWindow, s.incomingWindow)
```

Comment:
log the decision to send a Flow frame.
this is where the remoteOutgoingWindow showed to be max int.
it then never reaches incomingWindow/2, and we stop sending flow frames, which stops the protocol in some cases